recipe/README.md

```diff
@@ -136,25 +136,56 @@ then:
 
   # make a dependency version constraint stricter
   - tighten_depends:
-      # package to pin stricter
+      # package to pin stricter. 
+      # Example `abc >=1.0,<2.0a0` with max_pin='x.x' becomes `abc >=1.0,<1.1.0a0`
       name: matplotlib  # this field can use the fnmatch glob syntax
       # you must give one of max_pin or upper_bound
       # optional way to specify the new maximum pin as 'x', 'x.x', etc.
       max_pin: 'x.x'
-      # optional way to specify upper bound explicitly
+      # [DEPRECATED] optional way to specify upper bound explicitly
       # do not use with `max_pin`
+      #
+      # DEPRECATION NOTE: use `max_pin` for a relative scheme, or use `change_depends` 
+      # instead of `tighten_depends` for an absolute scheme
       upper_bound: 2.0.1
 
   # make a dependency version constraint looser
   - loosen_depends:
       # package to pin looser
+      # Example `abc >=1.0,<1.1.0a0` with max_pin='x.x' becomes `abc >=1.0,<2.0a0`
       name: matplotlib  # this field can use the fnmatch glob syntax
       # you must give one of max_pin or upper_bound
       # optional pinning expression 'x', 'x.x', etc. to set how much looser to make the pin
       max_pin: 'x.x'
-      # optional way to specify upper bound explicitly
+      # [DEPRECATED] optional way to specify upper bound explicitly
       # do not use with `max_pin`
+      #
+      # DEPRECATION NOTE: use `max_pin` for a relative scheme, or use `change_depends` 
+      # instead of `loosen_depends` for an absolute scheme
       upper_bound: 2.0.1
+  - change_depends:
```

Comment:
This name is too generic. Every operation in this repo changes something. Let's call this "bound_depends" or similar.